pkgs/by-name/ur/uradvd/package.nix

```
@@ -2,17 +2,25 @@
   stdenv,
   lib,
   fetchFromGitHub,
+  git,
+  testers,
+  uradvd,
 }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation (finalAttrs: {
   pname = "uradvd";
-  version = "0-unstable-2025-08-16";
+  version = "r26-1e64364d";
+
+  nativeBuildInputs = [
+    git
```

```suggestion
    gitMinimal
```
`gitMinimal` should be enough here.